bench.py

```diff
@@ -1,567 +0,0 @@
-            'rows': data.shape[0],
-            'columns': data.shape[1]
-        }
-    }
-    result['algorithm_parameters'] = {}
-    if alg_instance is not None:
-        if 'Booster' in str(type(alg_instance)):
-            alg_instance_params = dict(alg_instance.attributes())
-        else:
-            alg_instance_params = dict(alg_instance.get_params())
-            if ('min_samples_split' in alg_instance_params
-                    and 'handle' in alg_instance_params):
-                alg_instance_params['dtype'] = str(
-                    alg_instance_params['dtype'])
-        result['algorithm_parameters'].update(alg_instance_params)
-    if alg_params is not None:
-        result['algorithm_parameters'].update(alg_params)
-    return result
-
-
-def print_output(library, algorithm, stages, params, functions,
-                 times, metric_type, metrics, data, alg_instance=None,
-                 alg_params=None):
-    if params.output_format == 'json':
-        output = []
-        for i, stage in enumerate(stages):
-            result = gen_basic_dict(library, algorithm, stage, params,
-                                    data[i], alg_instance, alg_params)
-            result.update({'time[s]': times[i]})
-            if isinstance(metric_type, str):
-                result.update({f'{metric_type}': np.float64(metrics[i])
-                              if isinstance(metrics[i], np.float32)
-                              else metrics[i]})
-            elif isinstance(metric_type, list):
-                for ind, val in enumerate(metric_type):
-                    if metrics[ind][i] is not None:
-                        result.update({f'{val}': np.float64(metrics[ind][i])
-                                      if isinstance(metrics[ind][i], np.float32)
-                                      else metrics[ind][i]})
-            if hasattr(params, 'n_classes'):
-                result['input_data'].update({'classes': params.n_classes})
-            if hasattr(params, 'n_clusters'):
-                if algorithm == 'kmeans':
-                    result['input_data'].update(
-                        {'n_clusters': params.n_clusters})
-                elif algorithm == 'dbscan':
-                    result.update({'n_clusters': params.n_clusters})
-            # replace non-string init with string for kmeans benchmarks
-            if alg_instance is not None:
-                if 'init' in result['algorithm_parameters'].keys():
-                    if not isinstance(result['algorithm_parameters']['init'], str):
-                        result['algorithm_parameters']['init'] = 'random'
-                result['algorithm_parameters'].pop('handle', None)
-            output.append(result)
-        print(json.dumps(output, indent=4))
-
-
-def run_with_context(params, function):
-    if params.device != 'none':
-        from daal4py.oneapi import sycl_context
-        with sycl_context(params.device):
```

Comment:
was this replaced by an equivalent? or how is running gpu benchmarks handled

Reply:
Defined in config like, for example in this way:
```json
{ "algorithm": { "device": "gpu" } }
```
or
```json
{ "algorithm": { "sklearnex_context": { "target_offload": "gpu" } } }
```